world/stat_checks/check_maker.py

```diff
@@ -88,6 +94,68 @@ def announce_to_room(self):
             self.roll_message, options={"roll": True}
         )
 
+    def announce_to_players(self):
+        """
+        Sends a private roll result message to specific players as well as
+        all staff at self.character's location.
+        """
+        self_list = (
+            "me",
+            "self",
+            str(self.character).lower(),
+            str(self.character.key).lower(),
+        )
+
+        # Adjust receiver list; this is who actually will receive it besides staff and the caller.
+        receiver_list = self.receivers.copy()
```

Comment:
if `self.receivers` is None here this will raise an AttributeError. Right now that won't happen since `announce_to_players` is only called in a case that also populates receivers, but it'd be nice to prevent that from happening if this was extended/changed in the future.